crates/test-case-macros/Cargo.toml

```diff
@@ -10,7 +10,6 @@ readme        = "../../README.md"
 license       = "MIT"
 repository    = "https://github.com/frondeus/test-case"
 documentation = "https://docs.rs/test-case"
-exclude       = ["tests/snapshots/**/*"]
```

Comment:
I think this was from copy-pasting the root `Cargo.toml`, but again lmk if missing something :)